<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>694</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>694</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>861</width>
            <height>201</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="title">
              <string>Left Eye Picture</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <widget class="QLabel" name="imageEyeLeft">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>411</width>
                <height>161</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>1000</height>
               </size>
              </property>
              <property name="text">
               <string>LeftEyeImage</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="title">
              <string>Right Eye Picture</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <widget class="QLabel" name="imageEyeRight">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>30</y>
                <width>411</width>
                <height>161</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>1000</width>
                <height>1000</height>
               </size>
              </property>
              <property name="text">
               <string>RightEyeImage</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Anaglyph Picture</string>
         </property>
         <widget class="QLabel" name="imageAnaglyph">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>30</y>
            <width>841</width>
            <height>431</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Options</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Left Eye Image</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Right Eye Image</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxGlasses">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <property name="currentText">
         <string>red / cyan</string>
        </property>
        <item>
         <property name="text">
          <string>red / cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red / blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red / green</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Glasses type:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonCrop">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Crop Final Anaglyph</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonBackground">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Add background</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonColor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>114</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Color</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonGenerate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate Anaglyph</string>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>400</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Offset X:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>440</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Offset Y:</string>
        </property>
       </widget>
       <widget class="QSlider" name="verticalSlider">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>460</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labelColor">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>258</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
